### DISCUSSION TITLE
Duda css

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#422] [Q&A]
Buenos días, ya terminé mi práctica de css, tengo una duda porque por el diseño que metí a los botones se quedaron como estáticos como si fueran imágenes, ¿Cómo podría cambiar eso para que se puedan oprimir, aunque claro no llevara a ninguna página?

**Comment by AngelCruzO** [comment]
Si hiciste tus botones con listas, tienen que contener una etiqueta <a>, como en el ejemplo:

![image](https://user-images.githubusercontent.com/88981492/158650038-be9a13c4-6e25-45ea-867a-7b03185646d9.png)

Si lo hiciste de otra forma hay que ver tu codigo, para ayudarte con el atributo correcto. Si lo que deseas es que se vean de otro color al pasar el mouse es con el atributo :hover

**Reply by GloriaCoralCerecedo** [reply]
Gracias !

**Reply by AngelCruzO** [reply]
Una disculpa, no vi que el ejemplo no se veia bien, ya lo corregi. Suerte

**Reply by GloriaCoralCerecedo** [reply]
Ntp, gracias

**Reply by GloriaCoralCerecedo** [reply]
Es que creo que porque está en un contenedor y porque idea otras propiedad si no lo dejaré así jaja

**Reply by AngelCruzO** [reply]
Si gustas, te paso mi repo y te guias

**Comment by doguedogue** [comment]
A veces es mejor usar links en lugar de botones, ya sabes, los anchor `<a href="">` y les das forma como botón con padding y asi, y creas efectos como hover etc.

https://developer.mozilla.org/es/docs/Web/CSS/:hover
https://developer.mozilla.org/es/docs/Web/CSS/:active

Los botones puedes usarlos en formularios. Que estén fijos los tuyos (mmmm, creo que solo no se ve el cambio de estado), igual checate el efecto con pseudoclase :hover y :activate, no se si los tome igual.

Felicidades por terminar 🚀 🚀🚀

**Reply by GloriaCoralCerecedo** [reply]
Gracias, eso haré !

**Reply by ChanLuis** [reply]
Oye amigo, gracias, quedo perfecto al cambiar de botón a link. Estoy haciendo la misma parte del botón azul del menu. Grax
